src/pages/Validator/StakingBar.tsx

```diff
@@ -48,8 +57,9 @@ export default function StakingBar({validator}: ValidatorStakingBarProps) {
           <ListItemText primary="? APT" secondary="Rewards generated" />
         </ListItem>
         <Divider orientation="vertical" flexItem variant="fullWidth"></Divider>
-        <Button>Stake</Button>
+        <Button onClick={handleClickOpen}>Stake</Button>
       </ListItem>
+      <Staking handleDialogClose={handleClose} isDialogOpen={open} />
```

Comment:
```suggestion
      <Staking onDialogClose={handleDialogOpen} isDialogOpen={open} />
```
preferred naming convention